packages/lime-system/files/usr/lib/lua/lime/utils.lua

```diff
@@ -154,4 +154,12 @@ function utils.is_installed(pkg)
 	return utils.file_exists('/usr/lib/opkg/info/'..pkg..'.control')
 end
 
+function tableConcat(t1,t2)
+	local _,i
+	for _,i in ipairs(t2) do
+		table.insert(t1,i)
```

Comment:
This actually discard the key, so it has a concatenation semantic only in case the two tables are array, otherwise i believe it would degenerate to unexpected behaviors.
It is probably better to rename the function to arrayConcat ?

Reply:
renamed